src/SearchInput/SearchInput.tsx

```diff
@@ -192,7 +192,7 @@ const StyledResultsContainer = styled.div<ResultsContainer>`
   ${({ absolutePosition }) => absolutePosition && 'position: absolute;'}
   width: 100%;
   visibility: ${({ show }) => (show ? 'visible' : 'hidden')};
-  ${({ outlined }) => outlined && 'left: 0px; top: 93%;'};
+  ${({ outlined }) => outlined && 'left: 0px; top: 90%;'};
```

Comment:
better hiding the bottom border corners

Reply:
Shouldn't we remove the border-radius at the bottom when the popover is open instead of doing this?